fix:  missing ApiVersion and Kind in pgbench manifest using --dry-run and updating doc. (#1088)
cloudnative-pg/cloudnative-pg: docs/src/benchmarking.md

Example usage:
```

> kubectl cnpg pgbench <cluster-name> --pgbench-job-name <pgbench-job> --db-name <db-name> -n <NAMESPACE> -- --time 30 --client 1 --jobs 1

Reviewer: ```suggestion
kubectl cnpg pgbench \
  -n <namespace> <cluster-name> \
  --pgbench-job-name <pgbench-job> \
  --db-name <db-name> \
  -- --time 30 --client 1 --jobs 1
```

```

> Example of how to run it against a Cluster named `cluster-example` and Namespace `pgbench`:
> ```

Reviewer: ```suggestion
Example of how to run it against a `Cluster` named `cluster-example` in the `pgbench` namespace:
```
```

> kubectl cnpg pgbench cluster-example --pgbench-job-name pgbench-job -n pgbench -- --time 30 --client 1 --jobs 1

Reviewer: ```suggestion
kubectl cnpg pgbench \
   -n pgbench cluster-example \
   --pgbench-job-name pgbench-job \
   -- --time 30 --client 1 --jobs 1
```

```

> Example of how to run it on an existing database by using the `--db-name` flag and Namespace `pgbench`:

Reviewer: ```suggestion
Example of how to run it on an existing database by using the `--db-name` flag and
the `pgbench` namespace:
```

```

> kubectl cnpg pgbench cluster-example --db-name pgbench --pgbench-job-name pgbench-job -n pgbench -- --time 30 --client 1 --jobs 1

Reviewer: ```suggestion
kubectl cnpg pgbench \
  -n pgbench cluster-example \
  --db-name pgbench \
  --pgbench-job-name pgbench-job \
  -- --time 30 --client 1 --jobs 1
```
